openeo/rest/connection.py

```diff
@@ -1566,7 +1566,7 @@ def download(
 
         if outputfile is not None:
             with Path(outputfile).open(mode="wb") as f:
-                for chunk in response.iter_content(chunk_size=None):
+                for chunk in response.iter_content(chunk_size=10000000):
```

Comment:
create a constant DEFAULT_DOWNLOAD_CHUNK_SIZE in connection.py for this
and reuse it in the other places too

Reply:
This causes an ImportError due to circular imports. I think this should be set in a config file but I think that this functionality is not fully developed in openeo yet?

I can prevent the circular imports by defining the constant before the openeo imports in connection.py but this seems like bad coding to me

Reply:
I'd define the constant in openeo/rest/__init__.py to circumvent circular import issues